webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const webpack = require('webpack');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const markdownLoader = path.resolve(__dirname, 'loaders', 'marked-loader.js');
+
+const config = {
+  entry: [
+    './src/duxli.scss',
+    // Tedious, but any markdown files need to be added here.
+    // @todo FUTURE - Automatically determine all markdown files in source.
```

Comment:
May want to add a comment to look into loading the additional files in `loaders/marked-loader.js`.

That would be a non-standard choice, but the loader isn't intended to be published so it might not be a horrible option to consider.